internal/invocation/stripe_executor_test.go

```diff
@@ -0,0 +1,139 @@
+package invocation
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/goleak"
+)
+
+type orderChecker struct {
+	*testing.T
```

Comment:
Embedding `T` in this struct feels wrong to me.

Reply:
Removed as suggested.